vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -152,3 +159,11 @@ def _make_page_404_layout():
             ],
             className="page_error_container",
         )
+
+    def _infer_image(self, basename: str):
+        valid_extensions = [".apng", ".avif", ".gif", ".jpeg", ".jpg", ".png", ".svg", ".webp"]
+        assets_folder = Path(dash.get_app().config.assets_folder)
```

Comment:
@nadijagraca this is how @AnnMarieW thought of getting `assets_folder` too, so unless you found a better way of doing it let's go for this.